google/protobuf/duration.proto

```diff
@@ -0,0 +1,104 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
The only other option (until first class support is available) is to create a shell script that does the copying of native protobuf files in preparation of running ```bazel build```.